<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>windowSetting_Dialog</class>
 <widget class="QDialog" name="windowSetting_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>窗口设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images/img.qrc">
    <normaloff>:/new/prefix1/myappico.ico</normaloff>:/new/prefix1/myappico.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>361</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>窗口设置：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>361</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>窗口配置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="lineEdit_5"/>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="lineEdit_6"/>
      </item>
      <item row="0" column="6">
       <widget class="QLineEdit" name="lineEdit_7"/>
      </item>
      <item row="0" column="7">
       <widget class="QLineEdit" name="lineEdit_8"/>
      </item>
      <item row="0" column="8">
       <widget class="QLineEdit" name="lineEdit_9"/>
      </item>
      <item row="0" column="9">
       <widget class="QLineEdit" name="lineEdit_10"/>
      </item>
      <item row="0" column="10">
       <widget class="QLineEdit" name="lineEdit_11"/>
      </item>
      <item row="0" column="11">
       <widget class="QLineEdit" name="lineEdit_12"/>
      </item>
      <item row="0" column="12">
       <widget class="QLineEdit" name="lineEdit_13"/>
      </item>
      <item row="0" column="13">
       <widget class="QLineEdit" name="lineEdit_14"/>
      </item>
      <item row="0" column="14">
       <widget class="QLineEdit" name="lineEdit_15"/>
      </item>
      <item row="0" column="15">
       <widget class="QLineEdit" name="lineEdit_16"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>361</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="ok_pushButton">
     <property name="text">
      <string>设置</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>361</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images/img.qrc"/>
 </resources>
 <connections/>
</ui>
